Science/iss-overhead-check.60m.py

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-# <xbar.title>ISS Overhead Check</xbar.title>
-# <xbar.version>v0.0.1</xbar.version>
-# <xbar.author>Thiago Paes</xbar.author>
-# <xbar.author.github>mrprompt</xbar.author.github>
-# <xbar.desc>Check International Space Station Pass Times Over !</xbar.desc>
-# <xbar.dependencies>python</xbar.dependencies>
-
-from time import localtime
-from requests import get
-
-LATITUDE = -27.591665  # Put your latitude here
-LONGITUDE = -48.589599  # Put your longitude here
-ALTITUDE = 12  # Put your altitude here
-
-
-def iss_overhead():
-    url = "http://api.open-notify.org/iss-pass.json?lat={}&lon={}&alt={}".format(LATITUDE, LONGITUDE, ALTITUDE)
```

Comment:
Looks like this API is now at http://api.open-notify.org/iss-now.json. Any chance of salvaging this plugin using it?

Reply:
I don't think so, it looks like the whole API would need to be implemented from https://github.com/open-notify/Open-Notify-API/blob/88c34b793c9b0b299c8e941fd477e12c0b974353/app.py#L108.

Reply:
@mrprompt 

Any interest in updating this plugin?